client/src/components/googleMap/googleMap.jsx

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Loader} from '@googlemaps/js-api-loader';
+import './googleMap.sass';
+
+/* eslint-disable import/no-unresolved */
+const config = require('../../../../server/config');
```

Comment:
This is very bad. This is importing all the secrets in the config and just dumping them into the browser. Of course the api key needs to end up here, but this is also going to dump the db connection string into here, which is not good.

`const { mapsAPIKey } = require('../../../../server/config');` (object destructuring) would be better, but this would probably also be better handled by a different file for client-side stuff. We shouldn't need to be mixing `cleint/` and `server/`

Reply:
Are you suggesting moving the map api key to a new file? Similar to `server/config` but instead `client/config`

These are lessons I need to learn - some best practices stuff that the experienced people tell you about! So your updated code - that one only pulls the `mapsAPIKey` value out of the config JSON?

Reply:
If its only going to be used by the client, then yes, a new file.

Id google "Object Destructuring" and give it a read. But basically yes. It will find the thing called `mapsAPIKey` in the config, and only grab that, and stick it in the variable `mapsAPIKey`